open-company/open-company-web
# Sign up refresh

Card: https://trello.com/c/V7ho9xTh

Related PRs:
- ~[Storage](https://github.com/open-company/open-company-storage/pull/238)~
- ~[Support](https://github.com/belucid/open-company-support/pull/2) (This one is independent, meaning it can be reviewed on its own)~

To test:
- signup with email
- check you see Why Carrot field in Stabejo (with the question in front of it)
- invite someone via email
- accept invite via email
- signup with Slack
- invite someone via Slack
- accept the invite via Slack
- signup with Google
- signup using the team link
- login with email
- login with Slack
- login with Google
- signup with already used email
- signup with wrong email (bad format)
- login with not existing email
- login with wrong email/password
- reset your password
- signup with email on mobile
- login with email on mobile
- signup with email from desktop app ([Staging app is ok too](https://github.com/open-company/open-company-web/releases/tag/untagged-3b1db1ef7d46bb94ded5))
- login with email from desktop app ([Staging app is ok too](https://github.com/open-company/open-company-web/releases/tag/untagged-3b1db1ef7d46bb94ded5))
